girder/girder_large_image/rest/large_image_resource.py

```diff
@@ -449,56 +449,100 @@ def _deleteCachedImages(self, spec, associatedImages=False, imageKey=None):
+        Description('Create large images for all items within a folder.')
+        .notes('Does not work for items with multiple files.')
+        .modelParam('id', 'The ID of the folder.', model=Folder,
+                    level=AccessType.WRITE, required=True)
+        .param('createJobs', 'Whether job(s) should be used to create each '
+               'large image. Select true to use a job if needed, false to '
+               'never use a job, and always to always use a job.',
+               required=False, default='false', dataType='string',
+               enum=['true', 'false', 'always'])
+        .param('localJobs', 'Whether the job(s) created should be local.',
                required=False, default=False, dataType='boolean')
-        .param('localJobs', 'Whether the job(s) created should be local.', required=False,
-               default=False, dataType='boolean')
-        .param('recurse', 'Whether child folders should be recursed.', required=False,
-               default=False, dataType='boolean')
+        .param('recurse', 'Whether child folders should be recursed.',
+               required=False, default=False, dataType='boolean')
+        .param('cancelJobs', 'Whether unfinished large image job(s) associated '
+               'with items in the folder should be canceled, then a new large '
+               'image created. If false, items with an unfinished large image '
+               'will be skipped.', required=False, default=False,
+               dataType='boolean')
+        .param('redoExisting', 'Whether existing large images should be removed and '
+               'recreated.', required=False, default=False, dataType='boolean')
         .errorResponse('ID was invalid.')
         .errorResponse('Write access was denied for the folder.', 403),
     )
     def createLargeImages(self, folder, params):
         user = self.getCurrentUser()
-        createJobs = 'always' if self.boolParam('force', params, default=False) else True
-        return self.createImagesRecurseOption(folder=folder, createJobs=createJobs, user=user,
-                                              recurse=params.get('recurse'),
-                                              localJobs=params.get('localJobs'))
-
-    def createImagesRecurseOption(self, folder, createJobs, user, recurse, localJobs):
+        createJobs = params.get('createJobs')
+        if createJobs in ['true', 'false']:
+            createJobs = self.boolParam('createJobs', params, default=False)
+        return self.createLargeImagesRecurse(folder=folder, user=user,
+                                             recurse=params.get('recurse'), createJobs=createJobs,
+                                             localJobs=params.get('localJobs'),
+                                             cancelJobs=params.get('cancelJobs'),
+                                             redo=params.get('redoExisting'))
```

Comment:
I think breaking the options apart as suggested above would allow you to simplify this.